benchmark_tests/test_tutorial_docs.py

```diff
@@ -0,0 +1,75 @@
+import re
+import shutil
+import pytest
+
+@pytest.fixture(scope="function")
```

Comment:
@pytest.fixture(autouse=True), in this way you do not need to parametrize the tests